locales/languages/en.json

```
@@ -2248,5 +2248,26 @@
       "be_careful": "Be careful",
       "error_enter_number": "Error: Enter only numbers"
     }
+  },
+  "restore_wallet": {
+    "restore_needed_title": "Restore needed",
+    "restore_needed_description": "Something has gone wrong but don’t worry! Let’s try to restore your wallet.",
+    "restore_needed_action": "Restore wallet"
+  },
+  "wallet_restored": {
+    "wallet_restored_title": "Your wallet is ready!",
+    "wallet_restored_action": "Continue to wallet",
+    "wallet_restored_description_part_one": "You may need to add some of your assets, networks, and settings again manually.\n\n",
+    "wallet_restored_description_part_two": "Take a moment to ",
+    "wallet_restored_description_link": "back up your Secret Recovery Phrase ",
```

The translations remove the trailing spaces at the end of the strings, it's better to remove it. 
```suggestion
    "wallet_restored_description_part_two": "Take a moment to",
    "wallet_restored_description_link": "back up your Secret Recovery Phrase",
```

---

I'll move the spacing into the JSX